apps/openmw/mwworld/weather.cpp

```diff
@@ -289,44 +294,54 @@ WeatherManager::WeatherManager(MWRender::RenderingManager* rendering, MWWorld::F
 
     //Weather
     Weather clear;
+    clear.mObstructsCelestialBodies = false;
```

Comment:
On a related note, I wonder what "Clouds Maximum Percent" is supposed to do. It doesn't seem to affect the cloud transparency as it does in OpenMW. Only thing I noticed is that when set to "0", the moon texture becomes garbage.